### PR TITLE
Remove never-implemented WebVR interfaces

### PR DESCRIPTION
**Pull request by bershanskiy**
#### Summary

Remove never-implemented WebVR interfaces.

#### Motivation

These interfaces were never implemented.

#### Supporting details

Apparently, these interfaces existed only within WebVR spec drafts and were never actually implemented.
 - [These interfaces within Firefox source code](https://github.com/mozilla/gecko-dev/blob/master/dom/webidl/VRDisplay.webidl) (please also see "History" of the file, including history prior to renaming)
 - [WebVR spec](https://immersive-web.github.io/webvr/spec/1.1/) (please see IDL and spec history too)

#### Related issues

https://github.com/mdn/browser-compat-data/pull/16557

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -9552,13 +9552,17 @@
 /en-US/docs/Web/API/VRDevice	/en-US/docs/Web/API/VRDisplay
 /en-US/docs/Web/API/VRDevice/deviceId	/en-US/docs/Web/API/VRDisplay/displayId
 /en-US/docs/Web/API/VRDevice/deviceName	/en-US/docs/Web/API/VRDisplay/displayName
-/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRDisplay/hardwareUnitId
+/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRFieldOfView
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9555]:
```suggestion
/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRDevice
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I think one of the redirect is wrong.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (bershanskiy) [reply files/en-us/_redirects.txt:9555]: You are right!

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/_redirects.txt [hunk 2]
```diff
@@ -9552,13 +9552,17 @@
 /en-US/docs/Web/API/VRDevice	/en-US/docs/Web/API/VRDisplay
 /en-US/docs/Web/API/VRDevice/deviceId	/en-US/docs/Web/API/VRDisplay/displayId
 /en-US/docs/Web/API/VRDevice/deviceName	/en-US/docs/Web/API/VRDisplay/displayName
-/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRDisplay/hardwareUnitId
+/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRDevice
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9555]: Sorry, there was an extra redirect…

```suggestion
/en-US/docs/Web/API/VRDevice/hardwareUnitId	/en-US/docs/Web/API/VRDisplay
```

**Pull request merged**